src/native/client_bootstrap.c

```diff
@@ -38,16 +38,16 @@
 
 struct shutdown_callback_data {
     JavaVM *jvm;
-    jobject java_client_bootstrap;
+    jweak weak_java_client_bootstrap;
```

Comment:
the ```weak_``` feels a bit overkill

Reply:
Maybe just like ```client_bootstrap_ref``` or just ```java_client_bootstrap```?

Reply:
I can return it to the way it was at this point.  Was more to let the compile catch any place I missed.